src/web/app/components/adding-question-panel/adding-question-panel.component.spec.ts

```diff
@@ -36,4 +38,13 @@ describe('AddingQuestionPanelComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should contain anchor element with corresponding search query', () => {
+    const firstQuestionType = Object.keys(component.FeedbackQuestionType)[0] as FeedbackQuestionType;
```

Comment:
Can we add an assertion statement to check that Object.keys(component.FeedbackQuestionType)[0] is not constant sum question type, in case any changes are made in the future to this file, to ensure that there is an actual corresponding search query

Reply:
That is good idea. Will add!